### PR TITLE
[no ticket][risk=no] Puppeteer: break up cohorts-create test into two tests

### PR DESCRIPTION
**Pull request by aweng98**
Breaks up single Cohort test: first test covers Create action, second test covers Duplicate action.
Test suites can run in parallel, reduces total runtime.
Test steps are not changed.

### CODE REVIEW

File: e2e/tests/cohorts/cohorts-duplicate.spec.ts [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+import {MenuOption, LinkText, ResourceCard} from 'app/text-labels';
+import CohortBuildPage from 'app/page/cohort-build-page';
+import WorkspaceDataPage from 'app/page/workspace-data-page';
+import {findOrCreateWorkspace, signInWithAccessToken} from 'utils/test-utils';
+import {waitForText, waitWhileLoading} from 'utils/waits-utils';
+import CohortActionsPage from 'app/page/cohort-actions-page';
+import {Ethnicity} from 'app/page/cohort-search-page';
+
+
+describe('Cohorts', () => {
```
Comment (calbach) [inline e2e/tests/cohorts/cohorts-duplicate.spec.ts:15]: Shouldn't the test suites have different, more specific, names?

**Review by calbach — APPROVED** [review]
